### ISSUE TITLE
The "metadata.json" file for the box 'viniciusfs/centos7' was not found

### ISSUE THREAD

**Issue by viniciusfs** [viniciusfs/packer-templates#1]
Message `The "metadata.json" file for the box 'viniciusfs/centos7' was not found` while trying to up a new machine under libvirt provider.

```
(oitobits.net) vinicius@aryeh:~/Code/oitobits.net $ vagrant plugin install vagrant-libvirt
Installing the 'vagrant-libvirt' plugin. This can take a few minutes...
Fetching: formatador-0.2.5.gem (100%)
Fetching: excon-0.54.0.gem (100%)
Fetching: fog-core-1.43.0.gem (100%)
Fetching: ruby-libvirt-0.7.0.gem (100%)
Building native extensions.  This could take a while...
Fetching: fog-xml-0.1.2.gem (100%)
Fetching: fog-json-1.0.2.gem (100%)
Fetching: fog-libvirt-0.3.0.gem (100%)
Fetching: vagrant-libvirt-0.0.37.gem (100%)
Installed the plugin 'vagrant-libvirt (0.0.37)'!
(oitobits.net) vinicius@aryeh:~/Code/oitobits.net $ vagrant up --provider libvirt
Ignoring ruby-libvirt-0.7.0 because its extensions are not built.  Try: gem pristine ruby-libvirt --version 0.7.0
Bringing machine 'default' up with 'libvirt' provider...
==> default: Box 'viniciusfs/centos7' could not be found. Attempting to find and install...
    default: Box Provider: libvirt
    default: Box Version: >= 0
==> default: Loading metadata for box 'viniciusfs/centos7'
    default: URL: https://atlas.hashicorp.com/viniciusfs/centos7
==> default: Adding box 'viniciusfs/centos7' (v0.0.2) for provider: libvirt
    default: Downloading: https://atlas.hashicorp.com/viniciusfs/boxes/centos7/versions/0.0.2/providers/libvirt.box
The "metadata.json" file for the box 'viniciusfs/centos7' was not found.
Boxes require this file in order for Vagrant to determine the
provider it was made for. If you made the box, please add a
"metadata.json" file to it. If someone else made the box, please
notify the box creator that the box is corrupt. Documentation for
box file format can be found at the URL below:

https://www.vagrantup.com/docs/boxes/format.html

```
**Vagrantfile**
```
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure("2") do |config|
  config.vm.box = "viniciusfs/centos7"
  config.vm.synced_folder ".", "/vagrant_", disabled: true
end

```

**Comment by viniciusfs** [comment]
**Vagrantfile**
```
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure("2") do |config|
  config.vm.box = "viniciusfs/centos7"
  config.vm.synced_folder "../data", "/vagrant_data", disabled: true
end

```

**Comment by viniciusfs** [comment]
Not happening when using local box image:

```
(oitobits.net) vinicius@aryeh:~/Code/oitobits.net $ vagrant up --provider libvirt
Ignoring ruby-libvirt-0.7.0 because its extensions are not built.  Try: gem pristine ruby-libvirt --version 0.7.0
Bringing machine 'default' up with 'libvirt' provider...
==> default: Box '../packer-templates/boxes/centos-7.3-x86_64-libvirt.box' could not be found. Attempting to find and install...
    default: Box Provider: libvirt
    default: Box Version: >= 0
==> default: Box file was not detected as metadata. Adding it directly...
==> default: Adding box '../packer-templates/boxes/centos-7.3-x86_64-libvirt.box' (v0) for provider: libvirt
    default: Unpacking necessary files from: file:///home/vinicius/Code/packer-templates/boxes/centos-7.3-x86_64-libvirt.box
==> default: Successfully added box '../packer-templates/boxes/centos-7.3-x86_64-libvirt.box' (v0) for 'libvirt'!
==> default: Creating image (snapshot of base box volume).
==> default: Creating domain with the following settings...
==> default:  -- Name:              oitobits.net_default
==> default:  -- Domain type:       kvm
==> default:  -- Cpus:              1
==> default:  -- Memory:            512M
==> default:  -- Management MAC:    
==> default:  -- Loader:            
==> default:  -- Base box:          ../packer-templates/boxes/centos-7.3-x86_64-libvirt.box
==> default:  -- Storage pool:      default
==> default:  -- Image:             /home/vinicius/VMs/oitobits.net_default.img (10G)
==> default:  -- Volume Cache:      default
==> default:  -- Kernel:            
==> default:  -- Initrd:            
==> default:  -- Graphics Type:     vnc
==> default:  -- Graphics Port:     5900
==> default:  -- Graphics IP:       127.0.0.1
==> default:  -- Graphics Password: Not defined
==> default:  -- Video Type:        cirrus
==> default:  -- Video VRAM:        9216
==> default:  -- Keymap:            en-us
==> default:  -- TPM Path:          
==> default:  -- INPUT:             type=mouse, bus=ps2
==> default:  -- Command line : 
==> default: Creating shared folders metadata...
==> default: Starting domain.
==> default: Waiting for domain to get an IP address...
==> default: Waiting for SSH to become available...
    default: 
    default: Vagrant insecure key detected. Vagrant will automatically replace
    default: this with a newly generated keypair for better security.
    default: 
    default: Inserting generated public key within guest...
    default: Removing insecure key from the guest if it's present...
    default: Key inserted! Disconnecting and reconnecting using new SSH key...
==> default: Configuring and enabling network interfaces...

```

**Vagrantfile**
```
# -*- mode: ruby -*-
# vi: set ft=ruby :

Vagrant.configure("2") do |config|
  #config.vm.box = "viniciusfs/centos7"
  config.vm.box = "../packer-templates/boxes/centos-7.3-x86_64-libvirt.box"
  config.vm.synced_folder ".", "/vagrant", disabled: true
end
```

**Comment by viniciusfs** [comment]
Same error when trying virtualbox image:

```
(oitobits.net) vinicius@aryeh:~/Code/oitobits.net $ vagrant init viniciusfs/centos7; vagrant up --provider virtualbox
Ignoring ruby-libvirt-0.7.0 because its extensions are not built.  Try: gem pristine ruby-libvirt --version 0.7.0
`Vagrantfile` already exists in this directory. Remove it before
running `vagrant init`.
Ignoring ruby-libvirt-0.7.0 because its extensions are not built.  Try: gem pristine ruby-libvirt --version 0.7.0
Bringing machine 'default' up with 'virtualbox' provider...
==> default: Box 'viniciusfs/centos7' could not be found. Attempting to find and install...
    default: Box Provider: virtualbox
    default: Box Version: >= 0
==> default: Loading metadata for box 'viniciusfs/centos7'
    default: URL: https://atlas.hashicorp.com/viniciusfs/centos7
==> default: Adding box 'viniciusfs/centos7' (v0.0.2) for provider: virtualbox
    default: Downloading: https://atlas.hashicorp.com/viniciusfs/boxes/centos7/versions/0.0.2/providers/virtualbox.box
==> default: Box download is resuming from prior download progress
The "metadata.json" file for the box 'viniciusfs/centos7' was not found.
Boxes require this file in order for Vagrant to determine the
provider it was made for. If you made the box, please add a
"metadata.json" file to it. If someone else made the box, please
notify the box creator that the box is corrupt. Documentation for
box file format can be found at the URL below:

https://www.vagrantup.com/docs/boxes/format.html
```

**Comment by viniciusfs** [comment]
Solved at commit cc0e409774d4dfb37ce0c10316172391ccde1381
Issue on template configuration for post-processors.
